src/main/java/uk/gov/ons/ssdc/responseoperations/endpoint/UserGroupMemberEndpoint.java

```diff
@@ -68,6 +72,21 @@ public void removeUserFromGroup(
     }
 
     userGroupMemberRepository.delete(userGroupMember);
+
+    log.with(
+            new RMLoggingDto(
+                true,
+                userEmail,
+                "REMOVED USER FROM GROUP",
+                String.format(
+                    "User %s was removed from group %s",
+                    userGroupMember.getUser().getEmail(), userGroupMember.getGroup().getName())))
+        .warn(
```

Comment:
This feels more like an `info` level of logging rather than a warning?

Reply:
doh, cheers dude - was playing around with something and forgot to change back